SharedProjects/BabylonExport.Entities/BabylonPBRBaseSimpleMaterial.cs

```diff
@@ -85,8 +90,29 @@ public BabylonPBRBaseSimpleMaterial(BabylonPBRBaseSimpleMaterial original) : bas
 
         [DataMember]
         public BabylonPBRClearCoat clearCoat { get; set; }
+        
+        [DataMember]
+        public BabylonPBRSheenConfiguration sheen { get; set; }
 
         [DataMember(EmitDefaultValue = false)]
         public bool _unlit { get; set; }
+
+
+        // WARNING -TEMPORARY, MUST BE SWITCHED WITH THE USE OF PBRMaterial
+        #region WARNING
+
+        // IOR
+        public float? indexOfRefraction { get; set; }
+
+        // SPECULAR
+        public float? metallicF0Factor { get; set; }
+        public BabylonTexture metallicReflectanceTexture { get; set; }
+        public float[] metallicReflectanceColor { get; set; } = WhiteColor();
+
+        // note this following property does NOT even exits into Babylon (vesrion 4.2.1)
+        public BabylonTexture metallicReflectanceColorTexture { get; set; }
+
+        public BabylonPBRSubSurfaceConfiguration subSurface { get; set; }
+        #endregion
```

Comment:
Do you need to fix this before merging?

Reply:
Not really, this is part of the process by step. following the discussion you had with seb, i do not know yet what is your choice about adding gltf support to simpleMaterial or  refactor the whole material export to target the huge PbrMaterial. the second options is a bit heavy, so i might wait for the first options until the last minute :-)

Reply:
Removed by last commit to targeting BabylonPBRMaterial